{"hierarchy":{"paths":[["doc:\/\/BrdgeAPI\/documentation\/BrdgeAPI","doc:\/\/BrdgeAPI\/documentation\/BrdgeAPI\/BrdgeNetworkError"]]},"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"urlError"},{"text":"(","kind":"text"},{"text":"URLError","kind":"typeIdentifier","preciseIdentifier":"s:10Foundation8URLErrorV"},{"text":")","kind":"text"}]}],"kind":"declarations"}],"abstract":[{"code":"URLError","type":"codeVoice"},{"text":" thrown by ","type":"text"},{"type":"codeVoice","code":"URLSession.data(for request: URLRequest)"},{"type":"text","text":"."},{"type":"text","text":" "},{"type":"text","text":"A common cause of this error, is due to the user not being connected to the internet."}],"sections":[],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/BrdgeAPI\/documentation\/BrdgeAPI\/BrdgeNetworkError\/urlError(_:)"},"schemaVersion":{"minor":3,"patch":0,"major":0},"kind":"symbol","metadata":{"modules":[{"name":"BrdgeAPI"}],"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"urlError"},{"text":"(","kind":"text"},{"kind":"typeIdentifier","text":"URLError","preciseIdentifier":"s:10Foundation8URLErrorV"},{"kind":"text","text":")"}],"roleHeading":"Case","title":"BrdgeNetworkError.urlError(_:)","role":"symbol","symbolKind":"case","externalID":"s:8BrdgeAPI0A12NetworkErrorO03urlD0yAC10Foundation8URLErrorVcACmF"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/brdgeapi\/brdgenetworkerror\/urlerror(_:)"]}],"references":{"doc://BrdgeAPI/documentation/BrdgeAPI/BrdgeNetworkError/urlError(_:)":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"urlError","kind":"identifier"},{"kind":"text","text":"("},{"text":"URLError","kind":"typeIdentifier","preciseIdentifier":"s:10Foundation8URLErrorV"},{"text":")","kind":"text"}],"title":"BrdgeNetworkError.urlError(_:)","url":"\/documentation\/brdgeapi\/brdgenetworkerror\/urlerror(_:)","type":"topic","abstract":[{"code":"URLError","type":"codeVoice"},{"type":"text","text":" thrown by "},{"type":"codeVoice","code":"URLSession.data(for request: URLRequest)"},{"type":"text","text":"."},{"type":"text","text":" "},{"text":"A common cause of this error, is due to the user not being connected to the internet.","type":"text"}],"role":"symbol","kind":"symbol","identifier":"doc:\/\/BrdgeAPI\/documentation\/BrdgeAPI\/BrdgeNetworkError\/urlError(_:)"},"doc://BrdgeAPI/documentation/BrdgeAPI/BrdgeNetworkError":{"url":"\/documentation\/brdgeapi\/brdgenetworkerror","title":"BrdgeNetworkError","kind":"symbol","role":"symbol","navigatorTitle":[{"text":"BrdgeNetworkError","kind":"identifier"}],"abstract":[{"type":"text","text":"This is an error related to network calls."}],"identifier":"doc:\/\/BrdgeAPI\/documentation\/BrdgeAPI\/BrdgeNetworkError","type":"topic","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"BrdgeNetworkError","kind":"identifier"}]},"doc://BrdgeAPI/documentation/BrdgeAPI":{"title":"BrdgeAPI","url":"\/documentation\/brdgeapi","kind":"symbol","role":"collection","identifier":"doc:\/\/BrdgeAPI\/documentation\/BrdgeAPI","abstract":[{"type":"text","text":"The BR-DGE iOS SDK provides a seamless integration with the BR-DGE APIs through a native interface."}],"type":"topic"}}}